src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -88,7 +91,13 @@ def send_incident_alert_notification(
     )
 
     # If a parent notification exists for this rule and action, then we can reply in a thread
-    if parent_notification_message is not None:
+    # However, if the feature is turned off, do not use threads
+    if (
+        parent_notification_message is not None
+        and organization_integration_has_metric_alerts_flag_enabled(
```

Comment:
`organization_integration_has_metric_alerts_flag_enabled` is basically a one line function and I think `organization_integration_has_feature_enabled(
        integration=integration,
        organization_id=organization_id,
        feature=SlackIntegration.ISSUE_ALERTS_THREAD_FLAG,
    )` is as clear. If it helps with circular imports calling it directly could be a solution.